lib/phoenix/router.ex

```diff
@@ -375,6 +382,7 @@ defmodule Phoenix.Router do
               Plug.Conn.WrapperError.reraise(unquote(conn), kind, reason)
           end
         end
+        def __pipeline__(conn, unquote(plug)), do: unquote(plug)(conn, [])
```

Comment:
Shouldn't we rather just make the pipeline function public? It doesn't hurt... :)